File: C/others/BELOVED_C__VS__PYTHON/count_to_3_billion.py
import time

init = 0.0
end = 0.0
used_time = 0.0

for i in range(0, 3):
    init = time.perf_counter()
    for j in range(0, 3000000000):
        pass
    end = time.perf_counter()
    print(f"#{i+1} run: {end - init}")
    used_time += end - init

used_time /= 3
print(f"Execution time (~average): {used_time}")

# Average execution time on 3 runs: 158.03950410466678 seconds.

File: C/others/BELOVED_C__VS__PYTHON/count_to_3_billion.c
#include <stdio.h>
#include <time.h>

int main(){
    clock_t init, end;
    double used_time = 0;

    for(unsigned short i = 0; i < 10; i++){
        init = clock();
        for(unsigned i = 0; i < 3000000000; i++);
        end = clock();
        printf("#%d run: %lf\n", i+1, ((double) (end - init)) / CLOCKS_PER_SEC);
        used_time += ((double) (end - init)) / CLOCKS_PER_SEC;
    }
    used_time /= 10;
    printf("\nExecution time (~average): %lf\n", used_time);
    return 0;
}
// Average execution time on 10 runs -> 6.598900 seconds.